tyk-headless/values.yaml

```diff
@@ -24,34 +24,47 @@ secrets:
   useSecretName: ""
 
 redis:
-  # The addrs value will allow you to set your Redis addresses. If you are
-  # using a redis cluster, you can list the endpoints of the redis instances
-  # or use the cluster configuration endpoint.
+  # The addrs value will allow you to set your Redis addresses. 
+  #
+  # If you are using Redis (e.g. Bitnami Redis at bitnami/redis) then enter single
+  # endpoint.  If using "Redis Sentinel" update the port number (typically 26379).
+  #
+  # If using a "Redis Cluster" (e.g. bitnami/redis-cluster), you can list 
+  # the endpoints of the redis instances or use the cluster configuration endpoint.
+  # 
   # Default value: redis.{{ .Release.Namespace }}.svc.cluster.local:6379
+  #
   # addrs:
-  #   - redis.tyk.svc.cluster.local:6379
```

Comment:
Can you please re-add this address? It is used for our simple charts https://tyk.io/docs/tyk-oss/ce-helm-chart/#evaluation-only-via-simple-redis-chart

Reply:
Thank you for sharing the context; I have added this is a specific example.